pynubank/nubank.py

```diff
@@ -45,9 +45,9 @@ def _password_auth(self, cpf: str, password: str):
 
     def _save_auth_data(self, auth_data: dict) -> None:
         self.client.set_header('Authorization', f'Bearer {auth_data["access_token"]}')
-        self.feed_url = auth_data['_links']['events']['href']
```

Comment:
Para garantir a compatibilidade retroativa e passar todos os testes com o mínimo de mudança possível, eu recomendo somente utilizar estas novas URLs caso o campo 'magniture' estiver dentro dos dados de autenticação. Desta forma ficaria algo assim:

`        if 'magnitude' in auth_data['_links']:
            self.feed_url = auth_data['_links']['magnitude']['href']
            self.bills_url = auth_data['_links']['savings_account']['href']
        else:
            self.feed_url = auth_data['_links']['events']['href']
            self.bills_url = auth_data['_links']['bills_summary']['href']
`

Reply:
É uma boa, vou dar uma alterada aqui, mais seguro do que supor que `events` não será utilizada em nenhuma situação.

Reply:
Sim, com certeza aqui comigo até o momento não saiu esta versão nova, sendo assim já quebraria na minha situação.